### PR TITLE
⚠️ warn users when starting a flow without templates on whatsapp

### PR DESCRIPTION
**Pull request by nicpottier**
Also warns if they ARE using templates but they are not there or not approved. 

Also also switches to new WhatsApp API version.

### CODE REVIEW

File: temba/contacts/fields.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def get_json(self, value):
 
 class OmniboxField(forms.Field):
     default_error_messages = {}
-    widget = OmniboxWidget(attrs={"class": "omni_widget", "style": "width:85%"})
+    widget = OmniboxWidget(attrs={"class": "omni_widget", "style": "width:90%"})
```
Comment (nicpottier) [inline temba/contacts/fields.py]: Tested this new width in a bunch of places and definitely helps.

File: temba/tests/base.py [hunk 2]
```diff
@@ -197,6 +197,20 @@ def get_action_json(self, flow, uuid):
                     return action
         self.fail("Couldn't find action with uuid %s" % uuid)
 
+    def get_goflow(self, filename, substitutions=None):
```
Comment (nicpottier) [inline temba/tests/base.py]: When we have real importing we can probably replace this but for the time being this allows a new version flow to be brought in and tested.

**Comment by nicpottier** [comment]
Screen Shot 2019-05-03 at 2 30 39 PM

File: media/test_flows/goflow/template_flow.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"uuid":"c07d368e-fe1d-469d-b267-bca02d17c3e3","name":"Template Flow","spec_version":"13.0.0","language":"eng","type":"messaging","revision":11,"expire_after_minutes":10080,"localization":{},"nodes":[{"uuid":"5bf98a89-d38d-40ba-8e5e-d54cad1777f1","actions":[{"attachments":[],"text":"This is my test template message.","type":"send_msg","quick_replies":[],"uuid":"b103d563-9320-40fe-8f96-c9d3fbe0262d","templating":{"template":{"uuid":"f712e05c-bbed-40f1-b3d9-671bb9b60775","name":"affirmation"},"variables":[]}}],"exits":[{"uuid":"29dd720b-00ed-4304-97b0-38c65bc38e30"}]}],"_ui":{"nodes":{"5bf98a89-d38d-40ba-8e5e-d54cad1777f1":{"position":{"left":20,"top":40},"type":"execute_actions"}}},"_dependencies":{},"_results":[],"_waiting_exits":[],"version":"13.0.0"}
```
Comment (rowanseymour) [inline media/test_flows/goflow/template_flow.json]: maybe just format this for convenience

File: media/test_flows/goflow/template_flow.json [hunk 4]
```diff
@@ -1 +1,51 @@
...
+      "5bf98a89-d38d-40ba-8e5e-d54cad1777f1": {
+        "position": {
+          "left": 20,
+          "top": 40
+        },
+        "type": "execute_actions"
+      }
+    }
+  },
+  "_dependencies": {},
```
Comment (rowanseymour) [inline media/test_flows/goflow/template_flow.json]: these probably shouldn't be here if they're not correct - i.e. they're only added after saving/validation

**Pull request merged**